### DISCUSSION TITLE
Hamza's submission

### DISCUSSION THREAD

**Discussion by Stellarhamza** [stellar/sorobanathon#147] [General]
0 - https://litemint.com/items/GBBD2HWDVHHE3UNWRYEVFCQXGGREJHJDF7RK37B5FYJVGAK2CVHUSRAQ/NFT
1 - https://litemint.com/items/GAA4CJYWBHLI5PDB3DFUKRHVODRAHWHC3M3OSCIY4HEHMXUCIGLDXZQV/NFT
3-  https://litemint.com/items/GA3JT4RYCD3DA4Y75T5YDRQNSVZRKUPWOKL7LKIXBDK3QAZFCWVQ3UGO/NFT
4-  https://litemint.com/items/GAD4W5KGGUT66SGUAPYIZ5GIEOR3UIRHK6JIXRLMUO6W7PZT7UZEEACF/NFT
5-  https://litemint.com/items/GAUAMRG7XR4KMZQAAGZGO3C7YKRPYAWFPQS3EBRVPCDHOIPJAAF6TK4I/NFT
6- https://litemint.com/items/GCU5VOQ4XF7O3T5DG2ZWBD52H4PEDMDMEFXR6B6YXYD3V7J5THAE7T52/NFT
7- https://litemint.com/items/GB3SZBZOHVDJIZABLRUIB4TDHK4EVGCE2HCDQX7WMC23NEHGDGXKHVJA/NFT

**Comment by Xicaaa** [comment]
super cool!

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Reply by Stellarhamza** [reply]
Submitted the form thanks alot for giving us a chance :)

**Reply by Saylessunless** [reply]
Thank you, merry Xmas

**Reply by Stellarhamza** [reply]
Hello stellarjhon there has been an error, i entered my MEMO on the form but i believe you missed it, i checked on stellar expert, Please add on my Memo:- 424820737 as i am using binance memo is required :)

Thank you!
